<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>IP地址</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ip">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>用户名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_usr">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>密码</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_pwd">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>12</height>
            </size>
           </property>
           <property name="text">
            <string>端口</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_port">
           <property name="minimumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTextEdit" name="textEdit_cmdlist"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_sendcmd">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>41</height>
           </size>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTextEdit" name="textEdit_showresult"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
